src/script/campaign/mod/vco-listeners.lua

```diff
@@ -28,7 +28,7 @@ local function is_faction_vassal_or_destroyed(player_faction, target_faction_key
 end
 
 local function is_character_level_x(character, level)
```

Comment:
I am always up for descriptive functions, but maybe this one is too obvious and small? Up to you.

Reply:
Oh no, I very much like my functions to be appropriately named (i.e. immediately obvious) and short. This is fine in my book!